recipes/libproperties/all/conanfile.py

```diff
@@ -0,0 +1,47 @@
+from conans import ConanFile, CMake, tools
+
+
+class LibpropertiesConan(ConanFile):
+    name = "libproperties"
+    version = "1.1.2"
+    license = "Apache-2.0 License"
```

Comment:
```suggestion
    license = "Apache-2.0"
```

Use short form: https://spdx.org/licenses/Apache-2.0.html